ocfweb/docs/views/servers.py

```diff
@@ -57,81 +56,90 @@ def has_munin(self):
         return self.type in ('hypervisor', 'vm', 'server', 'desktop')
 
 
+def is_hidden(host):
+    return host['cn'][0].startswith('hozer-') or host['cn'][0].startswith('dev-')
```

Comment:
Again, this also needs to check whether a host is a staff VM (`(type=staffvm)`) and hide it if so.

Reply:
Looks like hosts that have type 'staffvm' are not included in the '(type=server)' query. Is it possible that each host only has one 'type' value? If that's the case, then maybe we don't need to filter staffvms.

Reply:
You're right about that, actually. Never mind.

Reply:
we don't really need to hide either of these types - they're useful for us as well to tell where the machines are located when we're trying to find them instead of guessing, sshing into every hypervisor, or going to puppetboard.